client/src/components/fortune.vue

```diff
@@ -1,41 +1,64 @@
 <template>
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-10">
-        <h1>
-            <img src="../assets/fortune-cookie.png" alt="fortune cookie" height="42" width="42">
-            Fortunes
-        </h1>
-        <hr><br><br>
-        <button type="button" class="btn btn-success btn-sm">Add Fortune</button>
-        <br><br>
-        <table class="table table-hover">
-          <thead>
-            <tr>
-              <th scope="col">Fortune</th>
-              <th scope="col">Author</th>
-              <th scope="col">Reviewed?</th>
-              <th></th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr v-for="(fortune, index) in fortunes" :key="index">
-              <td>{{fortune.fortune}}</td>
-              <td>{{fortune.author}}</td>
-              <td>
-                <span v-if="fortune.approved">Yes</span>
-                <span v-else>No</span>
-              </td>
-              <td>
-                <button type="button" class="btn btn-info btn-sm">Edit</button>
-                <button type="button" class="btn btn-danger btn-sm">Delete</button>
-              </td>
-            </tr>
-          </tbody>
-        </table>
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-10">
+          <h1>
+              <img src="../assets/fortune-cookie.png" alt="fortune cookie" height="42" width="42">
+              Fortunes
+          </h1>
+          <hr><br><br>
+          <button type="button" class="btn btn-success btn-sm" v-b-modal.fortune-modal>Add Fortune</button>
+          <br><br>
+          <table class="table table-hover">
+            <thead>
+              <tr>
+                <th scope="col">Fortune</th>
+                <th scope="col">Author</th>
+                <th scope="col">Reviewed?</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="(fortune, index) in fortunes" :key="index">
+                <td>{{fortune.fortune}}</td>
+                <td>{{fortune.author}}</td>
+                <td>
+                  <span v-if="fortune.approved">Yes</span>
+                  <span v-else>No</span>
+                </td>
+                <td>
+                  <button type="button"
+                          class="btn btn-info btn-sm"
+                          @click="editFortune(fortune)">Edit</button>
+                  <button type="button" class="btn btn-danger btn-sm">Delete</button>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
       </div>
-    </div>
+      <b-modal ref="addFortuneModal"
```

Comment:
Might want to consider making a modal component instead of having one massive template with everything in it.